hil-test/tests/lcd_cam_i8080.rs

```diff
@@ -89,4 +101,240 @@ mod tests {
             .unwrap();
         xfer.wait().unwrap();
     }
+
+    #[test]
+    fn test_i8080_8bit_is_seen_by_pcnt(ctx: Context<'static>) {
+        let cs_signal = ctx.io.pins.gpio8;
+        let unit0_signal = ctx.io.pins.gpio11;
+        let unit1_signal = ctx.io.pins.gpio12;
+        let unit2_signal = ctx.io.pins.gpio16;
+        let unit3_signal = ctx.io.pins.gpio17;
+
+        let pcnt = ctx.pcnt;
+
+        let unit_ctrl = cs_signal.peripheral_input();
+        let unit0_input = unit0_signal.peripheral_input();
+        let unit1_input = unit1_signal.peripheral_input();
+        let unit2_input = unit2_signal.peripheral_input();
+        let unit3_input = unit3_signal.peripheral_input();
+
+        pcnt.unit0
+            .channel0
+            .set_ctrl_mode(CtrlMode::Keep, CtrlMode::Disable);
+        pcnt.unit1
+            .channel0
+            .set_ctrl_mode(CtrlMode::Keep, CtrlMode::Disable);
+        pcnt.unit2
+            .channel0
+            .set_ctrl_mode(CtrlMode::Keep, CtrlMode::Disable);
+        pcnt.unit3
+            .channel0
+            .set_ctrl_mode(CtrlMode::Keep, CtrlMode::Disable);
+
+        pcnt.unit0
+            .channel0
+            .set_input_mode(EdgeMode::Hold, EdgeMode::Increment);
+        pcnt.unit1
+            .channel0
+            .set_input_mode(EdgeMode::Hold, EdgeMode::Increment);
+        pcnt.unit2
+            .channel0
+            .set_input_mode(EdgeMode::Hold, EdgeMode::Increment);
+        pcnt.unit3
+            .channel0
+            .set_input_mode(EdgeMode::Hold, EdgeMode::Increment);
+
+        let channel = ctx.dma.channel0.configure(false, DmaPriority::Priority0);
+        let pins = TxEightBits::new(
+            unit0_signal,
+            unit1_signal,
+            unit2_signal,
+            unit3_signal,
+            NoPin,
+            NoPin,
+            NoPin,
+            NoPin,
```

Comment:
Do you think we should also run the test for the upper 4 bits, (and the other 3 stripes in the 16-bit test)? This is already probably enough, but if it isn't terribly hard to loop through a few options with this implementation, I'd appreciate covering the other pins as well.

Reply:
I wanted to do this initially but it was rather tedious to do with GPIO output config wiping out the input settings. I'll happily add a loop once we figure that stuff out